pkg/test/case.go

```diff
@@ -386,6 +386,10 @@ func (t *Case) determineNamespace() *namespace {
 		ns.Name = fmt.Sprintf("kuttl-test-%s", petname.Generate(2, "-"))
 		ns.AutoCreated = true
 	}
+
+	if _, ok := os.LookupEnv("NAMESPACE"); ok { // user can supply this environment variable
```

Comment:
should this be 
```suggestion
	if _, ok := os.LookupEnv("NAMESPACE"); !ok { // user can supply this environment variable
```
otherwise the user's existing NAMESPACE environment variable will be overridden...

Reply:
lgtm